src/qmapshack/poi/CRawPoi.cpp

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************************************
+    Copyright (C) 2021 Henri Hornburg <pingurus@t-online.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#include "gis/Poi.h"
+#include "poi/CRawPoi.h"
+
+#include <QRegularExpression>
+
+CRawPoi::CRawPoi(QStringList data, QPointF coordinates, quint64 key, const QString& category)
```

Comment:
These can be all const references. No need to copy them in the stack and a second time into the member variables,